app/templates/components/global-search-tags.hbs

```diff
@@ -0,0 +1,17 @@
+{{#if @tags}}
+  <div class="global-search-tags">
+    {{#each @tags as |tag|}}
+      <span class="global-search-tag">
+        {{#if (eq tag "id")}}
```

Comment:
isn't this the same as the last `concat`ed option?

Reply:
Yes.